index.html

```diff
@@ -289,6 +289,38 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="#" class="form" method="get">
+          <input
+            name="Name"
+            id="Name"
+            type="text"
+            placeholder="FullName"
+            required
+            maxlength="30"
+          /><br />
+          <input
+            name="Email"
+            id="email"
+            type="email"
+            required
+            placeholder=""
+          /><br />
+          <textarea
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            placeholder=""
+          ></textarea
```

Comment:
Kindly add ```required```,  ```text max length``` and ```placeholder``` to this input, as it is part of the requirements for this project as well as improves the user experience :smiley: